crates/holochain/src/conductor/conductor.rs

```diff
@@ -3101,19 +3105,23 @@ async fn p2p_event_task(
                     }
                 }
 
-                match start {
-                    Some(start) => {
-                        let el = start.elapsed();
-                        let us = el.as_micros() as u64;
-                        let max_us = max_time
-                            .fetch_max(us, std::sync::atomic::Ordering::Relaxed)
-                            .max(us);
+                if current_num_tasks + 1 >= NUM_PARALLEL_EVTS {
```

Comment:
Yes the `+ 1` is accounting for the load returning the previous value. Because the ordering is Relaxed we might see values above the limit occasionally. It's not absolutely critical to report those so `==` would be valid. Just using `>` would be less correct because the value isn't supposed to exceed `NUM_PARALLEL_EVTS ` and that will only happen if there's some weak synchronization of the atomic value. I've moved the `+ 1` but I'm going to leave the comparison as it was for now